tree/ntuple/v7/doc/specifications.md

```diff
@@ -686,6 +686,10 @@ Every inner item (that describes a page) has the following structure:
 ```
 
 Followed by a locator for the page.
+Note that locators for byte ranges in a file do not need to reference pairwise distinct byte ranges.
+For instance, identical pages can point to the same page range.
+The DAOS locators, however, are guaranteed to point to unique object locations.
```

Comment:
Is this going to be true "forever" or is it just a current implementation limit? Maybe we are over-specifying this detail?

Reply:
I removed the DAOS comment.